lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -220,13 +312,23 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
                 redcap_record_instance, patient_reference, initial_encounter_reference)
 
             if is_complete('week_followup', redcap_record_instance):
-                follow_up_encounter_entry, follow_up_encounter_reference = create_follow_up_encounter(
-                    redcap_record_instance, patient_reference, site_reference, initial_encounter_reference)
+                # Don't set locations because the f/u survey doesn't ask for home address.
+                follow_up_encounter_entry, follow_up_encounter_reference = create_encounter(
+                    encounter_date = redcap_record_instance['fu_timestamp'].split()[0],
+                    patient_reference = patient_reference,
+                    site_reference = site_reference,
+                    collection_code = CollectionCode.HOME_HEALTH,
+                    parent_encounter_reference = initial_encounter_reference,
+                    encounter_reason_code = follow_up_encounter_reason_code(),
+                    encounter_identifier_suffix = "_follow_up",
+                    system_identifier = INTERNAL_SYSTEM,
+                    record = redcap_record_instance)
+
                 follow_up_questionnaire_entry = create_follow_up_questionnaire_response(
                 redcap_record_instance, patient_reference, follow_up_encounter_reference)
                 follow_up_computed_questionnaire_entry = create_computed_questionnaire_response(
                 redcap_record_instance, patient_reference, follow_up_encounter_reference,
-                birthdate, datetime.strptime(follow_up_encounter_entry['resource']['period']['start'], '%Y-%m-%d'))
+                birthdate, parse_date_from_string(follow_up_encounter_entry['resource']['period']['start']))
```

Comment:
This line (and the ones immediately above it) are under-indented, making this part of the code harder to read.

Reply:
Fixed in "FIXUP: Kairsten's feedback" commit.